cpp/src/decisiontree/batched-levelalgo/builder.cuh

```diff
@@ -205,6 +206,7 @@ struct Builder {
           IdxT nclasses,
           std::shared_ptr<const rmm::device_uvector<DataT>> quantiles)
     : handle(handle),
+      builder_stream(handle.get_internal_stream(treeid)),
```

Comment:
This would not work, since we do not have `n_strems = n_estimators`. We need to pass stream from random forest layer.

Reply:
Just pushed the change to pass the stream from RF all the way to Builder, let me know if that looks better

Reply:
Looks perfect now. Thanks Dante.